packages/grid/_modules_/grid/components/AutoSizer.tsx

```diff
@@ -119,10 +119,12 @@ export const AutoSizer = React.forwardRef<HTMLDivElement, AutoSizerProps>(functi
 
     const detectElementResize = createDetectElementResize(nonce, win);
     detectElementResize.addResizeListener(parentElement.current, handleResize);
+    window.addEventListener('resize', handleResize);
```

Comment:
1. We can't use the global window, we need to listen to the current window. (ownerWindow)
2. I don't think that it should be necessary. The resize event fired in the test is useless, proof: #535.

```suggestion
```

Reply:
It's due to an issue with the test.

Reply:
Why would we need to listen to resize? The logic uses a hidden scrollbar to detect the change of the container, he shouldn't be concerned with the size of the window.

Reply:
yes firing the event might be useless but it seems that if you want the resize to be triggered in the test, you need to bind the resize event, you should try from my branch...